examples/logical_enclosures.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.exceptions import HPOneViewException
+from config_loader import try_load_from_file
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+logical_enclosure_uri = config['logical_enclosure_uri']
```

Comment:
I am not comfortable with the 'logical_enclosure_uri' on the config file.  But it is a manner to force the user first configure this, to then run the example. So I found what the Ruby's team did and I think it's a great idea! The first  example is the 'get all', then from the result of the 'get all' they get the first item to pass to the next examples (They use the logical enclosure name. In your case you could use the logical enclosure uri). 
What do you think about it?

The code: https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/examples/logical_enclosure.rb